pytgcalls/pytgcalls/mtproto/telethon_bridge.py

```diff
@@ -137,21 +137,27 @@ async def edit_group_call_member(self, peer, volume: int = None, muted=False):
         self.client._handle_update(response)
 
     async def get_and_set_self_peer(self):
-        self.my_peer = await self.client.get_me()
+        self.my_peer = await self.client.get_me(input_peer=True)
 
         return self.my_peer
 
     async def get_and_set_group_call(self, group):
+
```

Comment:
pls format you code using black

Reply:
Sometimes I'm loosing hope in you guys and your code formatters but ok. Will do later in 1 hour